### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by abdulmaalik** [floatinghotpot/cordova-plugin-chartboost#2]
when i tried to install plugin using following cli 
cordova plugin add cordova-plugin-chartboost
it gives the following error

Fetching plugin "cordova-plugin-chartboost" via npm
Failed to fetch plugin cordova-plugin-chartboost via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: Registry returned 404 for GET on https://registry.npmjs.org/cordova-plugin-chartboost
can you please guide me how to install this plugin for chartboost ad . thanks

**Comment by ghost** [comment]
yes i am getting that as well ..i was unable to find this plugin in the npm registry doing a manual search..

You can do a manual install with

cordova plugin add https://github.com/floatinghotpot/cordova-plugin-chartboost

**Comment by vintage** [comment]
@floatinghotpot Why the plugin is not available through the npm anymore? I know that it's not a big deal, as the plugin works great without it, but newcomers can be confused that

`cordova plugin add cordova-plugin-chartboost`

doesn't work. Let's update the README maybe?
